### PR TITLE
Update players.yaml

### PR DESCRIPTION
**Pull request by ripvanwinkleee**
Top 200 player

### CODE REVIEW

**Comment by simonsan** [comment]
2021-09-12 21_37_27-Window

That graph looks a bit suspicious to me, to be honest. Do you have an explanation for the frequent MMR gain? @ripvanwinkleee

**Comment by ripvanwinkleee** [comment]
I played 1v1 with different accounts for a long time. I also play team games all the time. After more than a year I decided to upgrade my 1v1 account as well. this is the situation

**Comment by simonsan** [comment]
> I played 1v1 with different accounts for a long time. I also play team games all the time. After more than a year I decided to upgrade my 1v1 account as well. this is the situation

Which other accounts do you use? You might add them to this list as well as they are being aliased to you as well then.

File: data/players.yaml [hunk 1]
```diff
@@ -3613,3 +3613,8 @@
   liquipedia: Valak
   twitch: https://www.twitch.tv/imvalak
   team: QueNaNina
+  - name: Ripvanwinkle
+  country: tr
+  platforms:
+    de:
+    - '1820205'
```
Comment (simonsan) [inline data/players.yaml:3620]: You can add your other accounts to the list here. Also if you played on Voobly you might add that as well.

**Comment by ripvanwinkleee** [comment]
my voobly account:  https://www.voobly.com/profile/view/133845
my aoe2 de account : https://www.aoe2.net/#profile-3853480

**Comment by ripvanwinkleee** [comment]
my aoe2de account: https://www.aoe2.net/#profile-3853480
my voobly account: https://www.voobly.com/profile/view/133845

File: data/players.yaml (hunk 1 above)
Comment (simonsan) [inline data/players.yaml:3620]:
```suggestion
    de:
    - '1820205'
    - '3853480'
    voobly:
    - '133845'
```

**Comment by simonsan** [comment]
Anything else to add @ripvanwinkleee ? Twitch account or something?

Also your name `Ripvanwinkle` not any other capitalisation? It will be shown like this on other community pages e.g. aoe2.net

**Comment by ripvanwinkleee** [comment]
Yes, my nickname is Ripvanwinkle. Nothing else. Thank you.

**Pull request merged**